src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
I think you're confusing things. Safari won't work, no need to test. It does not support SW yet end of story. Reading the code, I think Chrome Android will break if you use a regular web notification (not via SW).